card/mod/05_standard/set/type/file.rb

```diff
@@ -126,7 +128,11 @@ def file_chooser args
             <span>
                 #{card.new_card? ? 'Add' : 'Replace'} #{card.attachment_name}...
             </span>
-             #{file_field card.attachment_name, class: 'file-upload slotter'}
+             <input class="file-upload slotter form-control" type="file" 
+                name="card[#{card.type_code}]" id="card_#{card.type_code}">
+             #{ hidden_field_tag 'attachment_type_id', card.type_id }
+             #{ hidden_field card.attachment_name, class: "attachment_card_name", value: '' }
```

Comment:
Line is too long. [93/80]<br>Space inside string interpolation detected.<br>Prefer single-quoted strings inside interpolations.